codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -369,7 +371,6 @@ final DefaultPropertyKey verifyKey(PropertyKey key) {
         private short weight = DEFAULT_PRIORITY_WEIGHT;
         private DefaultStream parent;
         private IntObjectMap<DefaultStream> children = IntCollections.emptyMap();
-        private int prioritizableForTree = 1;
```

Comment:
this was the count of how many nodes rooted at the current node are not closed, and in effect are allowed to be in the priority tree by their own right. Previously a closed node may remain in priority tree if it had decedents that were not closed. The last priority algorithm didn't necessarily rely on this but its a feature that would complicate this fix and I don't believe it is necessary.